vector/src/main/java/im/vector/app/features/settings/devices/v2/list/SessionsListHeaderView.kt

```diff
@@ -54,7 +54,12 @@ class SessionsListHeaderView @JvmOverloads constructor(
 
     private fun setTitle(typedArray: TypedArray) {
         val title = typedArray.getString(R.styleable.SessionsListHeaderView_devicesListHeaderTitle)
-        binding.sessionsListHeaderTitle.text = title
+        if (title.isNullOrEmpty()) {
```

Comment:
There is an extension method setTextOrHide() that could be used here to simplify reading:
`binding.sessionsListHeaderTitle.setTextOrHide(title)`

Reply:
Done.